catroid/src/main/java/org/catrobat/catroid/transfers/GoogleLoginHandler.java

```diff
@@ -47,56 +41,44 @@
 import org.catrobat.catroid.utils.DeviceSettingsProvider;
 import org.catrobat.catroid.utils.ToastUtil;
 
-public class GooglePlusLoginHandler implements GoogleApiClient.ConnectionCallbacks,
-		GoogleApiClient.OnConnectionFailedListener,
-		CheckOAuthTokenTask.OnCheckOAuthTokenCompleteListener,
+import static com.google.android.gms.auth.api.signin.GoogleSignIn.getClient;
+import static com.google.android.gms.auth.api.signin.GoogleSignIn.getSignedInAccountFromIntent;
+
+public class GoogleLoginHandler implements CheckOAuthTokenTask.OnCheckOAuthTokenCompleteListener,
 		GoogleLogInTask.OnGoogleServerLogInCompleteListener,
 		CheckEmailAvailableTask.OnCheckEmailAvailableCompleteListener,
 		GoogleExchangeCodeTask.OnGoogleExchangeCodeCompleteListener {
 
 	private AppCompatActivity activity;
-	public static final int GPLUS_REQUEST_CODE_SIGN_IN = 0;
 	public static final int REQUEST_CODE_GOOGLE_PLUS_SIGNIN = 100;
```

Comment:
When renaming from GOOGLE PLUS to only GOOGLE, please refactor also this variables.